crawler/src/main/java/com/movie/recommender/crawler/config/RabbitMQConfig.java

```diff
@@ -0,0 +1,28 @@
+package com.movie.recommender.crawler.config;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.amqp.core.*;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Slf4j
+@Configuration
+public class RabbitMQConfig {
+
+    public static final String QUEUE_NAME = "location.update.queue";
```

Comment:
message sent when a new location is added
update naming to reflect this use case
```
QUEUE_NAME -> NEW_LOCATION_QUEUE = "new.location.queue"
EXCHANGE_NAME -> NEW_LOCATION_EXCHANGE = "new.location.exchange"
```